runtime/src/integration/working_group.rs

```diff
@@ -184,3 +185,91 @@ impl<T: stake::Trait + working_group::Trait<GatewayWorkingGroupInstance>>
         remaining_imbalance
     }
 }
+
+pub struct OperationsWgStakingEventsHandler2<T> {
```

Comment:
Please, remove code duplication. Using rust macro could be one of the possible solutions.